doc/release/new-policy.rst

```diff
@@ -0,0 +1,341 @@
+        The intended support time for each series is at least two years since the first release.
+
+        At the moment when this document is published, there are two release series: ``1.10`` and ``2.x.y``.
+
+    Release version
+
+        Release version is a Tarantool distribution which is thoroughly tested and ready for production usage.
+        It is bound to a certain commit.
+        Release version label consists of three numbers:
+
+        ..  code-block:: text
+
+            MAJOR.MINOR.PATCH
+
+These numbers correspond to the three types of release versions:
+
+..  glossary::
+
+    Major release
+
+        Major release is the first :term:`release version <release version>` of its own
+        :term:`release series <release series>`.
+        It introduces new features and can have a few backward-incompatible changes.
+        Such release changes the first version number:
+
+        ..  code-block:: text
+
+            MAJOR.0.0
+
+            3.0.0
+
+    Minor release
+
+        Minor release introduces a few new features, but guarantees backward compatibility.
+        There can be a few bugs fixed as well.
+        Such release changes the second version number:
+
+        ..  code-block:: text
+
+            MAJOR.MINOR.0
+
+            3.1.0
+            3.2.0
+
+    Patch release
+
+        Patch release fixes bugs from an earlier release, but doesn't introduce new features.
+        Such release changes the third version number:
+
+        ..  code-block:: text
+
+            MAJOR.MINOR.PATCH
+
+            3.0.1
+            3.0.2
+
+Release versions conform to a set of requirements:
+
+    *   The release has gone through pre-release testing and adoption
+        in the internal projects until there were no doubts regarding its stability.
+
+    *   There are no known bugs in the typical usage scenarios.
+
+    *   There are no degradations from the previous release or release series, in case of a major release.
+
+Backwards compatibility is guaranteed between all versions in the same release series.
+It is also appreciated, but not guaranteed between different release series (major number changes).
+A detailed description of compatibility guarantees will be published later.
+
+Pre-release versions
+~~~~~~~~~~~~~~~~~~~~
+
+..  glossary::
+
+    Pre-release version
+
+        Pre-release versions are the ones published for testing and evaluation,
+        and not intended for production use.
+        Such versions use the same pattern with an additional suffix:
+
+        ..  code-block:: text
+
+            MAJOR.MINOR.PATCH-suffix
+
+There are a few types of pre-release versions:
+
+..  glossary::
+
+    Nightly build
+
+        Nightly builds reflect the state of current development process.
+        They're used entirely for development and testing,
+        and not intended for any external use.
+
+        Nightly builds have suffixes made with ``git describe --always --long-dev``:
```

Comment:
It should be `$(git describe --always --long)-dev` or `` `git describe --always --long`-dev `` assuming `2.10.2-149-g1575f3c07-dev` and so on. We can eliminate the git describe command and just give the example, because it is the user documentation.